### PR TITLE
[DRAFT] ERC-4902: Decentralized Autonomous Access (DAA)

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by eartho-group** [comment]
Join to our discord for chat with us
https://discord.gg/5QbuTNTG2q

**Review by MicahZoltu — COMMENTED** [review]
Please see https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md for an example of how to layout/format an EIP.  The header section is incredibly strict, and the top level sections should all exactly match the template.  Within sections there is a bit more leeway.

**Comment by eartho-group** [comment]
Yes i know, but im waiting to number allocation from them, only then ill be able to change it

File: EIPS/eip-daa [hunk 1]
```diff
@@ -0,0 +1,131 @@
+---
+eip: eip-daa
```
Comment (MicahZoltu) [inline EIPS/eip-daa:2]:
```suggestion
eip: 4902
```

File: EIPS/eip-4902.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
+---
+eip: 4902
+title: Decentralized Autonomous Access (DAA)
+description: DAAs represent an access over digital or physical things & resources, with the ability to connect with entities that seeking to use it
```
Comment (lightclient) [inline EIPS/eip-4902.md:4]:
```suggestion
description: DAAs represent access to digital or physical things, with the ability to connect with entities seeking to use them
```
grammar errors

File: EIPS/eip-4902.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+author: <Daniel @eartho-group>
+discussions-to: https://github.com/ethereum/eips/issues/4902
```
Comment (lightclient) [inline EIPS/eip-4902.md:6]: Please open a thread on eth magicians for this proposal. This is a requirement to merge to draft.

File: EIPS/eip-4902.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-22
+requires: 20
+---
+
+## Abstract
+As humans, we are constantly seeking as much access to resources as possible, We also make access trades for our needs regularly, it can be for information, food, rides, content, hotels, jobs, and so on. The trade could be as simple as me looking for access to food and the other side looking for access to money.
```
Comment (lightclient) [inline EIPS/eip-4902.md:15]: Several grammar errors here. Also, please do not use first or second person pronouns in EIP text.

File: EIPS/eip-4902.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+author: <Daniel @eartho-group>
+discussions-to: https://github.com/ethereum/eips/issues/4902
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-03-22
+requires: 20
+---
+
+## Abstract
```
Comment (lightclient) [inline EIPS/eip-4902.md:14]: Abstract should be relatively short, maybe 1 paragraph.

File: EIPS/eip-4902.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+
+## Motivation
+Because ownership is based on social consensus and relies on a third party to record and enforce it, it is nearly impossible to transition from the current economy to the web3 world, because that data cannot be shared and cannot be on-chain.
+
+By changing our mindset from myth of “ownership, control, assets” to “access”, We are able to eliminate several issues such as the need for trust on the old registrations and records of the institutions.
+
+Trading access is a convenient and cost-effective way to use resources without the financial, emotional, oracle issues, or social burdens that come with ownership.
+
+## Specification
```
Comment (lightclient) [inline EIPS/eip-4902.md:41]: Please use markdown code blocks for code sections of the spec.

File: EIPS/eip-4902.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+Test cases for an implementation are mandatory for EIPs that are affecting consensus changes.  If the test suite is too large to reasonably be included inline, then consider adding it as one or more files in `../assets/eip-####/`.
+
+## Reference Implementation
+In private repo for now, so will be open
+
+## Security Considerations
+Every action should be accepted by the actors.
+
+## Copyright
+Eartho Group, Daniel, Tel Aviv
```
Comment (lightclient) [inline EIPS/eip-4902.md:131]: The copyright must be CC0 as specified in EIP-1. Please review EIP-1 for the exact text.

**Review by lightclient — CHANGES_REQUESTED** [review]

File: EIPS/eip-4902.md (hunk 4 above)
Comment (eartho-group) [reply EIPS/eip-4902.md]: im not a native speaker, i used serval tools but its probably still not perfect. maybe you can give it your touch

File: EIPS/eip-4902.md (hunk 5 above)
Comment (eartho-group) [reply EIPS/eip-4902.md]: its really important to me be able to deliver the concept. how do you suggest to do so without the background?

Comment (MicahZoltu) [reply EIPS/eip-4902.md]: Abstract should be a high level human readable description of the specification.

Motivation should describe *why* this EIP is a good idea, but it doesn't need to be exhaustive.

Specification should only describe the specification itself, not its reasoning/motivation.

Rationale should describe why you made specific decisions within the specification (like why a particular value was chosen).

----

Something to keep in mind, EIPs are specifications, they are not meant to be exhaustive documents on a particular topic.  If you want to discuss a lot about the background, reasoning, history, etc. you should put that in a blog, website, docs site, etc. and leave the EIP just as the specification.

**Pull request closed without merging**